torch/csrc/jit/passes/quantization.cpp

```diff
@@ -588,8 +620,8 @@ void QuantizeHelper::quantizeTensors() {
     dequant = insertQuantDeQuantCall(v, qparams, scalar_type);
     v->replaceAllUsesWith(dequant->output());
     Node* q = dequant->input(0)->node();
-    // replaceAllUsesWith rewrote all uses of V, but we want to keep one: the one
-    // used in quant node. Restore it here:
+    // replaceAllUsesWith rewrote all uses of V, but we want to keep one: the
+    // one used in quant node. Restore it here:
```

Comment:
why do we get this back? I remember it is already properly handled in the insertQuantDequantCall.

Reply:
maybe I'll need to rebase